ember-flight-icons/tests/dummy/app/templates/index.hbs

```diff
@@ -1,11 +1,62 @@
 {{page-title "Index"}}
+<h1 class="ds-h1">
+  Flight Icons
+</h1>
+{{!-- TODO this can't go from an h1 to an h3 --}}
+<div class="rounded-lg mb-8 bg-gray-200 overflow-hidden shadow divide-y divide-gray-200 sm:divide-y-0 sm:grid sm:grid-cols-2 sm:gap-px">
+  {{!-- TODO import our colors into the tailwind config so we can replace "ring-indigo-500" --}}
+  <div class="rounded-tl-lg rounded-tr-lg sm:rounded-tr-none relative group bg-white p-6 focus-within:ring-2 focus-within:ring-inset focus-within:ring-indigo-500">
+    <h3 class="ds-h3 mb-2">
+      <LinkTo @route="design" class="ds-a focus:outline-none">
+        <span class="absolute inset-0" aria-hidden="true"></span>
+        <FlightIcon @name="path" @size="24" @color="var(--brand)" class="sm:mr-0 md:mr-2 lg:mr-4" /> Getting Started for Design
+      </LinkTo>
+    </h3>
+    <p class="ds-p">
+      Bring Flight into your Figma files, contribute a new icon, or learn about design best practices.
+    </p>
+  </div>
+
+  <div class="sm:rounded-tr-lg relative group bg-white p-6 focus-within:ring-2 focus-within:ring-inset focus-within:ring-indigo-500">
+    <h3 class="ds-h3 mb-2">
+        <LinkTo @route="engineering" class="ds-a focus:outline-none">
+        <!-- Extend touch target to entire panel -->
+        <span class="absolute inset-0" aria-hidden="true"></span>
+        <FlightIcon @name="code" @size="24" @color="var(--brand)" class="sm:mr-0 md:mr-2 lg:mr-4" /> Getting Started For Engineers
+      </LinkTo>
+    </h3>
+    <p class="ds-p mt-2">
+      Learn how to implement Flight icons into your project with options for direct SVG support or as an Ember addon.    </p>
+  </div>
+
+  <div class="relative group bg-white p-6 focus-within:ring-2 focus-within:ring-inset focus-within:ring-indigo-500">
+    <h3 class="ds-h3">
+        <a href="" class="ds-a focus:outline-none">
+        <!-- Extend touch target to entire panel -->
+        <span class="absolute inset-0" aria-hidden="true"></span>
+        <FlightIcon @name="edit" @size="24" @color="var(--brand)" class="sm:mr-0 md:mr-2 lg:mr-4" /> Contributing Guide
```

Comment:
Do we know what this will eventually link to?

Reply:
yes, I will add it 👍